app/client-v2/apps/goat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@p4b/goat",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
I think there is a bug in syncpack when workspace is enabled, this doesn't look right that it is changing the canonical package version. A fix will come in the next release.

Reply:
Thanks @JamieMason